addons/ingress-controller/resources/ingress-controller.cue

```diff
@@ -0,0 +1,12 @@
+output: {
```

Comment:
Why add the ingress controller?  Before we supported optional modules, the addon was as atomic as possible and the Ingress controller was present in most clusters.

Reply:
It is possible to separate the Ingress Controller as an Addon.

Reply:
If user's cluster doesn't have an ingress-controller he/she can choose to install this addon. Just an option for user.

Reply:
But users can not make select.

Reply:
Doc will tell user if cluster hasn't a ingress-controller user can choose this addon.

Reply:
origin: https://github.com/kubevela/kubevela.io/pull/694#discussion_r893023507